src/views/sponsorship.html

```diff
@@ -0,0 +1,302 @@
+                                <p class="shown">
+                                    Whether you’re a young company looking to increase your reputation or a large firm with a desire to hidden off your capabilities to future tech trendsetters, Hack Cambridge brings together the perfect audience for any company: the next generation of innovators and thought-leaders. Few messages about
+                                </p>
+                                <p class="hidden">
+                                    Whether you’re a young company looking to increase your reputation or a large firm with a desire to show off your capabilities to future tech trendsetters, Hack Cambridge brings together the perfect audience for any company: the next generation of innovators and thought-leaders. Few messages about your brand and culture resonate louder with passionate hackers than to be there, stuck in alongside them during 24 intense hours of innovation. From product demos to sponsored messages, we want to help you maximise your exposure and recognition among the people who will drive the next generation of tech growth.
+    
+                                </p>
+                            </div>
+                            <div class="column">
+                                <p class="shown">
+                                    your brand and culture resonate louder with passionate hackers than to be there, stuck in alongside them during 24 intense hours of innovation. From product demos to sponsored messages, we want to help you maximise your exposure and recognition among the people who will drive the next generation of tech growth.
+                                </p>
+                                <h4>
+                                    Product Adoption and Live Testing
+                                </h4>
+                                <p>
+                                    Show off your new product or demo your flagship hero service, and let the hackers at it for the ultimate test! Hackers love to experiment. They will test the robustness, functionality, market fit, and user-friendliness of any system presented to them in clever and unexpected ways. You’ll be sure to leave with invaluable feedback on how to improve your product and lasting mindshare amongst upcoming leaders.
+                                </p>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="half unit row shown"></div>
+                    <div id="sponsorship-university-of-cambridge-endorsement" class="half unit row centred">
+                        <object id="sponsorship-university-of-cambridge-logo-object" data="{{ asset('images/hc-2018/university-of-cambridge-logo.svg') }}"></object>
+                        <p id="sponsorship-university-of-cambridge-text" style="margin:0">
+                            The University of Cambridge<br>endorses Hack Cambridge.
```

Comment:
nit: space between Cambridge and endorses for screenreader? (throughout template)

Reply:
Tested some of these with Voiceover and it appears a line break is treated as whitespace—is that not typical?

Reply:
Oh, nice — no worries then 🙂